front/src/modules/activities/tasks/components/PageAddTaskButton.tsx

```diff
@@ -4,8 +4,14 @@ import { useOpenCreateActivityDrawer } from '@/activities/hooks/useOpenCreateAct
 import { PageAddButton } from '@/ui/layout/page/PageAddButton';
 import { useFilter } from '@/ui/object/object-filter-dropdown/hooks/useFilter';
 
-export const PageAddTaskButton = () => {
-  const { selectedFilter } = useFilter();
+type PageAddTaskButtonProps = {
+  filterId: string;
+};
+
+export const PageAddTaskButton = ({ filterId }: PageAddTaskButtonProps) => {
+  const { selectedFilter } = useFilter({
+    filterScopeId: filterId,
```

Comment:
let's hide the "scope" from component API: 
- useFilterDropdown({filterDropdownId: filterDropdownId})